integration_tests/data/pipeline_out/ingest_unit_test_parser_input/TESTCCLW.executive.4.4.json

```diff
@@ -8,14 +8,20 @@
   "document_md5_sum": null,
   "document_slug": "european-union_2013_decision-no-13862013eu-of-the-european-parliament-and-of-the-council-of-20-november-2013-on-a-general-union-environment-action-programme-to-2020-living-well-within-the-limits-of-our-planet_8570_3017",
   "document_metadata": {
+    "collection_summary": null,
+    "collection_title": null,
+    "corpus_type_name": null,
+    "corpus_import_id": null,
+    "document_title": null,
+    "geographies": null,
     "name": "DECISION No 1386/2013/EU OF THE EUROPEAN PARLIAMENT AND OF THE COUNCIL of 20 November 2013 on a General Union Environment Action Programme to 2020 \u2018Living well, within the limits of our planet\u2019",
     "description": "description",
     "import_id": "TESTCCLW.executive.4.4",
     "slug": "european-union_2013_decision-no-13862013eu-of-the-european-parliament-and-of-the-council-of-20-november-2013-on-a-general-union-environment-action-programme-to-2020-living-well-within-the-limits-of-our-planet_8570_3017",
     "family_import_id": "TESTCCLW.family.4.0",
     "family_slug": "slug_TESTCCLW.family.4.0",
     "publication_ts": "2013-01-01T00:00:00",
-    "date": "01/01/2013",
```

Comment:
Why are dates null?

Reply:
@olaughter: Do you have any theories about why this would be?

Reply:
Oh, yep this was me 😁 https://github.com/climatepolicyradar/cpr-sdk/pull/81/

Reply:
The field remains with the intention of being backward compatible, but is no longer being calculated. As part of that PR I looked for instances of us using `date` it seems we haven't cared about it since opensearch ¯\_(ツ)_/¯